cpp/src/bindings/bindings_vec2d.cpp

```diff
@@ -0,0 +1,45 @@
+#include "jni_refs.hpp"
+#include "helpers/general.hpp"
+#include "rive/math/vec2d.hpp"
+#include <jni.h>
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+	// AABB
+
+	JNIEXPORT jlong JNICALL Java_app_rive_runtime_kotlin_core_Vec2D_constructor(
```

Comment:
curious: why wrap this at all? Seems like a lot to just pass around 2 floats. Can we use Android's point (or just an array)?